NumberBox: Add min & max API Clarity (#9080)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxNumberBox/1 Configuration/min.md

---
---
##### shortDescription

> The minimum value NumberBox allows users to enter. Does not affect programmatic value changes.

Reviewer: ```suggestion
The minimum value users can enter in the NumberBox. This option does not affect values assigned programmatically.
```

Author: I believe it is a slight bit more accurate to say that this property affects value changes rather than values themselves. Moreover, "programmatically" is slightly more complex. I propose the following combination of the original text and your suggestion:
```suggestion
The minimum value users can enter in the NumberBox. This option does not affect programmatic value changes.
```
What do you think?
